example/lib/main.dart

```diff
@@ -43,39 +45,42 @@ class MyApp extends StatelessWidget {
             ListTile(
               title: const Text('Add normal event'),
               trailing: const Icon(Icons.calendar_today),
-              onTap: () {
-                Add2Calendar.addEvent2Cal(
+              onTap: () async {
+                final result = await Add2Calendar.addEvent2Cal(
                   buildEvent(),
                 );
+                print(result);
```

Comment:
Should we remove these prints?

Reply:
I left them intentionally to illustrate the result-feature that didn't exist before